odd-platform-ui/src/components/Management/LabelsList/LabelsListSkeleton/LabelsListSkeleton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Skeleton from '@material-ui/lab/Skeleton';
+import { Grid, withStyles } from '@material-ui/core';
+import { styles, StylesType } from './LabelsListSkeletonStyles';
+
+interface SkeletonProps extends StylesType {
+  length?: number;
+}
+
+const LabelsListSkeleton: React.FC<SkeletonProps> = ({
+  classes,
+  length,
+}) => {
+  const randomSkeletonWidth = () => {
```

Comment:
This function is being used in all skeleton components. Would be nice to take it out and reuse across all of them.